lib/posts/20160923_ramda_di.md

```diff
@@ -151,7 +151,7 @@ easy to initialize all of them in a single function call with `applySpec()`.
 ```javascript
 co(function * () {
   const db = yield mongodb.MongoClient.connect(process.env.MONGO_URL);
-  const User = applySpec(require('./user')(db);
+  const User = applySpec(require('./user')(db));
```

Comment:
You're right that I'm missing parentheses, but the extra parentheses should be before `(db)` because applySpec() returns a function if you pass it one arg

Reply:
Oh. I've just copied it from the other line similar to it, right above [Moving on](https://github.com/vkarpov15/thecodebarbarian.com/blob/master/lib/posts/20160923_ramda_di.md#moving-on). I'll delete the PR, feel free to make any necessary changes.